app/search_builders/sufia/my_shares_search_builder.rb

```diff
@@ -9,5 +9,8 @@ def show_only_shared_files(solr_parameters)
     solr_parameters[:fq] += [
       "-" + ActiveFedora::SolrQueryBuilder.construct_query_for_rel(depositor: scope.current_user.user_key)
     ]
+
+    return unless Flipflop.enable_mediated_deposit?
```

Comment:
This method used to return what was returned by:

```ruby
solr_parameters[:fq] += [
        "-" + ActiveFedora::SolrQueryBuilder.construct_query_for_rel(depositor: scope.current_user.user_key)
      ]
```

and now it returns `nil` if mediated deposit is off. Would this work instead:

```ruby
solr_parameters[:fq] += ['-suppressed_bsi:true'] if Flipflop.enable_mediated_deposit?
```

?